app/views/decidim/participations/admin/participations/_moderated.html.erb

```diff
@@ -0,0 +1,49 @@
+<div class="card-section">
+  <div class="table-scroll">
+    <table class="table-list">
+      <thead>
+        <tr>
+          <th>
+            <%= sort_link(query, :published_on, t("models.participation.published_on", scope: "decidim.participations"), default_order: :desc ) %>
+          </th>
+          <th>
+            <%= sort_link(query, :type, t("models.participation.fields.type", scope: "decidim.participations") ) %>
+          </th>
+          <th>
+            <%= t("models.participation.fields.title", scope: "decidim.participations")%>
+          </th>
+          <th>
+            <%= sort_link(query, :status, t("models.participation.status", scope: "decidim.participations") ) %>
+          </th>
+
+
+          <th class="actions"><%= t("actions.title", scope: "decidim.participations") %></th>
+        </tr>
+      </thead>
+      <tbody>
+        <% participations.each do |participation| %>
+          <tr data-id="<%= participation.id %>">
+            <td>
+              TODO
+            </td>
+            <td>
+              TODO
+            </td>
+            <td>
+              <%= participation.title %><br />
+            </td>
+            <td>
+              TODO
+            </td>
+            <td class="table-list__actions">
+            <!-- TODO path des boutons -->
```

Comment:
is it to be done later? If no remove comment. 
Should it reference a github issue so we can quickly work on it later?

Reply:
This is specify in corresponding issues